src/modules/csv-export/v1/csv-export.controller.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Controller,
+  Get,
+  Param,
+  ParseIntPipe,
+  Post,
+  Query,
+} from '@nestjs/common';
+import {
+  ApiAcceptedResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiQuery,
+  ApiTags,
+} from '@nestjs/swagger';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import { object } from 'zod';
+import { CsvExportService } from '@/modules/csv-export/v1/csv-export.service';
+import { Job } from 'bullmq';
+import {
+  JobStatusDto,
+  JobStatusErrorDto,
+  JobStatusResponseDto,
+} from '@/routes/jobs/entities/job-status.dto';
+
+//TODO create a separate path in swagger
+@ApiTags('export')
+@Controller({
+  path: 'export',
+  version: '1',
+})
+export class CsvExportController {
+  constructor(private readonly csvExportService: CsvExportService) {}
+
+  @ApiAcceptedResponse({ type: object }) //TODO fix (job-related data) + status 200/201 or 202 ?
+  @ApiQuery({ name: 'executionDateGte', required: false, type: String })
+  @ApiQuery({ name: 'executionDateLte', required: false, type: String })
+  @ApiQuery({ name: 'limit', required: false, type: Number })
+  @ApiQuery({ name: 'offset', required: false, type: Number })
+  @Post('chains/:chainId/:safeAddress')
+  async export(
```

Comment:
nitpick: to be more self-explanatory, I would call it `launch ExportJob` or something around this.